inscriptions/ISic030002.xml

```diff
@@ -0,0 +1,412 @@
+                        <ptr target="http://zotero.org/groups/382445/items/F2GG87EQ"/>
+                    </bibl>
+                    <bibl>
+                        <author>Bravo</author>
+                        <date>1992</date>
+                        <citedRange>71-75</citedRange>
+                        <ptr target="http://zotero.org/groups/382445/items/GKSSRUX7"/>
+                    </bibl>
+                    <bibl type="corpus" n="LSAG[2]">
+                        <author>Jeffery, Johnston</author>
+                        <date>1990</date>
+                        <citedRange>459, no. 23a</citedRange>
+                        <ptr target="http://zotero.org/groups/382445/items/QUWGZA84"/>
+                    </bibl>
+                    <bibl type="corpus" n="IGDS1">
+                        <author>Dubois</author>
+                        <date>1989</date>
+                        <citedRange>219</citedRange>
+                        <ptr target="http://zotero.org/groups/382445/items/RWXNUFEX"/>
+                    </bibl>
+                    <bibl>
+                        <author>Manganaro</author>
+                        <date>1965</date>
+                        <citedRange>194-195</citedRange>
+                        <ptr target="http://zotero.org/groups/382445/items/RGTHAFKT"/>
+                    </bibl>
+                    <bibl type="corpus" n="LSAG">
+                        <author>Jeffery</author>
+                        <date>1961</date>
+                        <citedRange>268, pl. 51; 276, no. 15</citedRange>
+                        <ptr target="http://zotero.org/groups/382445/items/V6CGSWPS"/>
+                    </bibl>
+                    <bibl type="bullettin" n="SEG">
+                        <citedRange>12.407 (Woodhead 1955)</citedRange>
+                        <ptr target="http://zotero.org/groups/382445/items/F2GG87EQ"/>
+                    </bibl>
+                    <bibl>
+                        <author>Guarducci</author>
+                        <date>1959-1960</date>
+                        <citedRange>254-259</citedRange>
+                        <ptr target="http://zotero.org/groups/382445/items/GPUF5NUH"/>
+                    </bibl>
+                    <bibl>
+                        <author>Pugliese Carratelli</author>
+                        <date>1956</date>
+                        <citedRange>151, pl. 34</citedRange>
+                        <ptr target="http://zotero.org/groups/382445/items/PJDRZH66"/>
+                    </bibl>
+                    <bibl>
+                        <author>Guarducci</author>
+                        <date>1949-1950 (printed in1952)</date>
+                        <citedRange>111-14, fig.5</citedRange>
+                        <ptr target="http://zotero.org/groups/382445/items/UG26WH3A"/>
+                    </bibl>
+                    <bibl type="bullettin" n="SEG">
```

Comment:
bulletin; 04.027; it would be better not to have the name and year afterwards if possible, to improve search/filter